test/e2e/session_test.go

```diff
@@ -3,201 +3,123 @@ package e2e_test
 import (
 	"fmt"
 	"math/rand"
-	"os"
+	"regexp"
+	"strings"
 	"time"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	y "github.com/onsi/ginkgo"
+	z "github.com/onsi/gomega"
 
-	"github.com/solo-io/solo-kit/pkg/api/v1/resources/core"
-	v1 "github.com/solo-io/squash/pkg/api/v1"
-	sqOpts "github.com/solo-io/squash/pkg/options"
-	squashcli "github.com/solo-io/squash/pkg/squashctl"
+	gokubeutils "github.com/solo-io/go-utils/kubeutils"
 	"github.com/solo-io/squash/test/testutils"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
-//const KubeEndpoint = "http://localhost:8001/api"
-
 func Must(err error) {
-	Expect(err).NotTo(HaveOccurred())
+	z.Expect(err).NotTo(z.HaveOccurred())
+}
+func check(err error) {
+	z.Expect(err).NotTo(z.HaveOccurred())
```

Comment:
must and check are doing the same thing. remove one of them? 
also, do `ExpectWithOffset(1,...` instead of `Expect`